grammars/SeedPython.g4

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 The Aha001 Team.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
```

Comment:
Apache license info included in the source code should be kept as it is, including the text wrapping and the empty lines. See "HOW TO APPLY THE APACHE LICENSE TO YOUR WORK" at https://www.apache.org/licenses/LICENSE-2.0 for the original lines.